### PR TITLE
49312-disable_codecov

### PR DESCRIPTION
**Pull request by anikolskiy**
https://miceportal.atlassian.net/browse/MP-49312

This PR should disable codecov globally in every application that is using workflowator.

### CODE REVIEW

**Review by stereosupersonic — COMMENTED** [review]
i think it's easier to remove the *code_coverage* line on each application's like here https://github.com/konvenit/katalysator/blob/master/.github/workflows/main.yml#L54
because if you change it here (workflowator), you have to tag it and update the applications anyway

**Review by Arsen7 — APPROVED** [review]

**Comment by anikolskiy** [comment]
@stereosupersonic do we actually need these versions? Can't it just be `@master` in each apps' `workflows/main.yml`? Was there any case when some apps were not updated almost immediately after `workflowator#master` was updated?

**Comment by stereosupersonic** [comment]
with Tags it's better to be backward compatible. you would not break all other workflows.

But even if you change it to master then you have to adjust each application. Still for me the best and explicit change would be to change `code_coverage: false` on each application.

File: .github/workflows/compose_run.yaml [hunk 1]
```diff
@@ -68,14 +68,16 @@ jobs:
...
-        docker-compose -f docker-compose.test.yml exec -T ${{ inputs.application-name }} ${{ inputs.cmd}}
+        docker-compose -f docker-compose.test.yml exec -T -e CI=true ${{ inputs.application-name }} ${{ inputs.cmd }}
 
     - name: Run command with code coverage
       if: ${{ inputs.code_coverage }}
       run: |
         # Could we download that file, review it, and then use that - approved by us - local copy?
-        ci_env=`bash <(curl -s https://codecov.io/env)`
-        docker-compose -f docker-compose.test.yml exec -T -e CI=true -e CODECOVERAGE=true $ci_env ${{ inputs.application-name }} ${{ inputs.cmd}}
+        # ci_env=`bash <(curl -s https://codecov.io/env)`
```
Comment (stereosupersonic) [inline .github/workflows/compose_run.yaml:77]: why you only commented it out and not remove it completely ?

Comment (anikolskiy) [reply .github/workflows/compose_run.yaml:77]: Explained in the comment: https://github.com/konvenit/workflowator/pull/8/files#diff-f9aaa882e25a94b1a4ddaab3655ff382acb11aa8f00366edb720acf6a61e9b2aR79 . "(probably temporarily)". But you're right. If we need to add another service, `ci_env` will probably not be needed or will be different.

**Comment by anikolskiy** [comment]
Ok @stereosupersonic , you convinced me to go the individual apps route. Closing this PR.

**Pull request closed without merging**